src/Tasks/Backup/Zip.php

```diff
@@ -97,13 +97,24 @@ public function add(string | iterable $files, string $nameInZip = null): self
             $files = [$files];
         }
 
+        $compressionMethod = config('backup.backup.destination.compression_method', null);
+        $compressionLevel = config('backup.backup.destination.compression_level', 9);
+
         foreach ($files as $file) {
             if (is_dir($file)) {
                 $this->zipFile->addEmptyDir(ltrim($nameInZip ?: $file, DIRECTORY_SEPARATOR));
             }
 
             if (is_file($file)) {
                 $this->zipFile->addFile($file, ltrim($nameInZip, DIRECTORY_SEPARATOR)).PHP_EOL;
```

Comment:
Hi, what `.PHP_EOL`  do?? it is ouside

Reply:
I was wondering as well what it's doing there, but it is in the main branch:
https://github.com/spatie/laravel-backup/blob/f81e8495a7944eb9684580d99f40bbdb4b3f7f93/src/Tasks/Backup/Zip.php#L106C88-L106C88

and I have decided to leave it untouched

Reply:
Most likely it was a typo.

Reply:
Ok, removed unnecessary `.PHP_EOL`. It's been sitting there since 2016 :)